src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -200,11 +200,11 @@ def handle_message(self, message):
                     "timestamp": contents["timestamp"],
                     "query_subscription_id": contents["subscription_id"],
                     "project_id": subscription.project_id,
-                    "subscription_dataset": subscription.dataset,
-                    "subscription_query": subscription.query,
-                    "subscription_aggregation": subscription.aggregation,
-                    "subscription_time_window": subscription.time_window,
-                    "subscription_resolution": subscription.resolution,
+                    "subscription_dataset": subscription.snuba_query.dataset,
+                    "subscription_query": subscription.snuba_query.query,
+                    "subscription_aggregation": subscription.snuba_query.aggregate,
```

Comment:
This will now be a string instead of 0/1?

Reply:
That's fine, this is just logs. This is actually more descriptive